bedrock/products/templates/products/vpn/landing.html

```diff
@@ -52,7 +52,7 @@
 {% endblock %}
 
 {% block sub_navigation %}
-  {% include 'products/vpn/includes/subnav.html' %}
+  {% include 'products/vpn/includes/subnav-refresh.html' %}
 {% endblock %}
```

Comment:
As it {% extends "products/vpn/base.html" %}, this block is already there:

https://github.com/mozilla/bedrock/blob/178aa6b762bff6b2741d5acd1e89f5d0e64ff1a7/bedrock/products/templates/products/vpn/base.html#L16-L18

right?

```suggestion
```